install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -110,7 +128,11 @@ spec:
           # The initial delay for the liveness probe is intentionally large to
           # avoid an endless kill & restart cycle if in the event that the initial
           # bootstrapping takes longer than expected.
+{{- if semverCompare ">=1.20-0" .Capabilities.KubeVersion.Version }}
+          initialDelaySeconds: 0
```

Comment:
also, is there any reason to keep initialDelaySeconds for `readinessProbe` ?

Reply:
No real reason, I just removed it for 1.20.